.gitignore

```diff
@@ -1,3 +1,5 @@
 lib/
 log/
 node_modules/
+._*
+.DS_Store
```

Comment:
Those are system/developer specific rules, not project specific ones, and they should be put in your global `.gitignore`, not here.